### PR TITLE
used css grid to make layout changes on StudentGroup component

### PR DESCRIPTION
**Pull request by SteveLorentzen**
need to refactor code and work on full screen
need to change logic to select student as soon as group opens

### CODE REVIEW

File: src/screens/StudentGroup.tsx [hunk 1]
```diff
@@ -92,23 +91,73 @@ const StudentGroup: React.FC = () => {
...
+          md: `'top top'
+                '. .'
+                'pic name'
+                'pic fact'
+                'pic bottom'`,
+          lg: `'top top top top'
+          '. . . . '
+              '. pic name .'
+              '. pic fact .'
+              '. pic bottom .'`,
```
Comment (trevorfehrman) [inline src/screens/StudentGroup.tsx]: try more descriptive names here i think, great idea though

File: src/screens/StudentGroup.tsx [hunk 2]
```diff
@@ -118,21 +167,49 @@ const StudentGroup: React.FC = () => {
...
+                as="h2"
+                fontSize={{ base: '1.1rem', md: '1.6rem', lg: '2rem' }}
+                color="var(--main-color-medium)"
+                width="100%"
+              >
+                {studentGroupDocument?.selectedStudent.studentInfo.selectedFact.title}
+              </Heading>
+
+              <Heading as="h2" fontSize={{ base: '1.5rem', md: '2.3rem', lg: '3rem' }} letterSpacing=".05em">
+                {studentGroupDocument?.selectedStudent.studentInfo.selectedFact.value}
```
Comment (trevorfehrman) [inline src/screens/StudentGroup.tsx]: `?` shouldn't be needed right? we can't get here if line 180 is falsey.

**Review by trevorfehrman — APPROVED** [review]

File: src/screens/StudentGroup.tsx (hunk 2 above)
Comment (SteveLorentzen) [reply src/screens/StudentGroup.tsx]: Good point, I'll get rid of that.

File: src/screens/StudentGroup.tsx (hunk 1 above)
Comment (SteveLorentzen) [reply src/screens/StudentGroup.tsx]: Thanks!  I find it really hard to name containers for multiple unrelated things.  One of the areas that I called "bottom" sits at the bottom of the screen but has two things in it: the student list and the next button.  Not sure how to stay concise enough for the grid-template-areas definition not to look god-awful but still be descriptive. I'll work on this and find a nice middle ground

**Pull request merged**